cmd/telemetry.go

```diff
@@ -27,6 +29,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const gaugeTelemetryEnabled = "GAUGE_TELEMETRY_ENABLED"
+
+var telemetryEnv = os.Getenv(gaugeTelemetryEnabled)
+
+func telemetryEnabled() bool {
+	if telemetryEnv == "" {
+		return config.TelemetryEnabled()
+	}
```

Comment:
Maybe this check can be removed. If `telemetryEnv` is empty `strconv.ParseBool` will return err which is handled in next check.